core/shared/src/main/scala/zio/stm/TRandom.scala

```diff
@@ -0,0 +1,368 @@
+package zio.stm
+
+import zio.random.Random
+import zio.{ BuildFrom, BuildFromOps, Chunk, ZLayer, random }
+
+object TRandom extends Serializable {
+
+  trait Service {
+    def nextBoolean: STM[Nothing, Boolean]
```

Comment:
Just a super-nitpick, I'd use `USTM` instead of `STM[Nothing, A]` wherever possible.